mava/components/jax/building/parameter_client.py

```diff
@@ -93,8 +93,11 @@ def on_building_executor_parameter_client(self, builder: SystemBuilder) -> None:
             builder: SystemBuilder.
         """
         # Create policy parameters
-        params = {}
-        get_keys = []
+        params: Dict[str, Any] = {}
+        # Executor does not explicitly set variables i.e. it adds to count variables
```

Comment:
Interesting that this pops up? I thought @AsadJeewa pushed this code to develop?

Reply:
I added the fix to this branch, not develop